jdaviz/cli.py

```diff
@@ -6,8 +6,6 @@
 import tempfile
 
 import click
-from voila.app import Voila
-from voila.configuration import VoilaConfiguration
 
 from jdaviz import __version__
 from jdaviz.app import _verbosity_levels
```

Comment:
how much do these two imports cost relative to voila?  Would moving them (especially the verbosity levels) to some standalone file speed up the imports by avoiding a full jdaviz import?

Reply:
While we can easily revert to hardcoding verbosity level options here, we cannot really not import `__version__` and you need that the moment you set up click or argparse for version reporting, which is baked into the setup (so you cannot import it in if/else). The moment you import jdaviz for anything in CLI mode, it takes like 30 secs.